bin/copyRawDataToTmp.sh

```diff
@@ -0,0 +1,370 @@
+	#
+	cat <<EOH
+===============================================================================================================
+Script to copy (sync) data from a succesfully finished run from tmp to prm storage.
+
+Usage:
+
+	$(basename "${0}") OPTIONS
+
+Options:
+
+	-h	Show this help.
+	-n	Dry-run: Do not perform actual sync, but only list changes instead.
+	-a	Archive mode: only copy the raw data to prm and do not split the flowcell based samplesheet
+		into project samplesheets to trigger the project-based analysis of the data with a subsequent pipeline.
+	-g [group]
+		Group for which to process data.
+	-w [workingDirectory]
+		working directory a.k.a. rootDir/groupDir/LFS_dir
+		e.g. /groups/umcg-gd/prm06 or /groups/umcg-gap/scr01
+	-s [samplesheetServerLocation]
+		location of the samplesheet 
+		e.g. localhost (default) or wingedhelix
+	-l [level]
+		Log level.
+		Must be one of TRACE, DEBUG, INFO (default), WARN, ERROR or FATAL.
+	-r [root]
+		Root dir on the server specified with -s and from where the raw data will be fetched (optional).
+		By default this is the SCR_ROOT_DIR variable, which is compiled from variables specified in the
+		<group>.cfg, <source_host>.cfg and sharedConfig.cfg config files (see below.)
+		You need to override SCR_ROOT_DIR when the data is to be fetched from a non default path,
+		which is for example the case when fetching data from another group.
+
+Config and dependencies:
+
+	This script needs 4 config files, which must be located in ${CFG_DIR}:
+		1. <group>.cfg       for the group specified with -g
+		2. <this_host>.cfg   for this server. E.g.: "${HOSTNAME_SHORT}.cfg"
+		3. <source_host>.cfg for the source server. E.g.: "<hostname>.cfg" (Short name without domain)
+		4. sharedConfig.cfg  for all groups and all servers.
+	In addition the library sharedFunctions.bash is required and this one must be located in ${LIB_DIR}.
+===============================================================================================================
+EOH
+	trap - EXIT
+	exit 0
+}
+
+#
+##
+### Main.
+##
+#
+
+#
+# Get commandline arguments.
+#
+log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Parsing commandline arguments ..."
+declare group=''
+declare dryrun=''
+while getopts ":g:l:s:r:ahn" opt
+do
+	case "${opt}" in
+		h)
+			showHelp
+			;;
+		g)
+			group="${OPTARG}"
+			;;
+		s)
+			samplesheetsServerLocation="${OPTARG}"
+			;;
+		n)
+			dryrun='-n'
+			;;
+		l)
+			l4b_log_level="${OPTARG^^}"
+			l4b_log_level_prio="${l4b_log_levels["${l4b_log_level}"]}"
+			;;
+		\?)
+			log4Bash 'FATAL' "${LINENO}" "${FUNCNAME[0]:-main}" '1' "Invalid option -${OPTARG}. Try $(basename "${0}") -h for help."
+			;;
+		:)
+			log4Bash 'FATAL' "${LINENO}" "${FUNCNAME[0]:-main}" '1' "Option -${OPTARG} requires an argument. Try $(basename "${0}") -h for help."
+			;;
+		*)
+			log4Bash 'FATAL' "${LINENO}" "${FUNCNAME[0]:-main}" '1' "Unhandled option. Try $(basename "${0}") -h for help."
+			;;
+	esac
+done
```

Comment:
The `-r` and `-a` options are not parsed

Reply:
In addition -n for dryrun argumented is parsed and documented in the commandline help, but not implemented.